specification/programmableconnectivity/Azure.ProgrammableConnectivity/apis/number.tsp

```diff
@@ -65,15 +65,16 @@ model NumberVerificationWithoutCodeContent {
   networkIdentifier: NetworkIdentifier;
 
   ...NumberDevice;
+
+  @doc("Redirect URI needed for auth callback.")
+  redirectUri: string;
```

Comment:
I *think* the type should be uri, not string - please check.

Reply:
yes, this will be something like `https://blah.blah/outauth/v2/...`

In the codebase I only see `url` as a type, is there also a `uri` type? I think we should just use `url` perhaps

Reply:
`url` - see https://typespec.io/docs/language-basics/built-in-types#string-types